composer.json

```diff
@@ -70,6 +70,9 @@
             },
             "drupal/entity_browser": {
                 "Provide inline entity form FieldWidgetDisplay": "https://www.drupal.org/files/issues/2858438_6.patch"
+            },
+            "drupal/scheduler": {
+                "Scheduler integration with core Content moderation": "https://www.drupal.org/files/issues/2018-07-31/2798689-98.patch"
```

Comment:
This is not the latest version of the patch you've made. The latest version seems to have considerable changes. Also I'm concerned about the implementation of the unpublished state in this patch because at the moment it lists both draft and unpublished in the thunder use-case. Draft is not going to un-publish anything. It's going to create a new forward revision exactly the same as the published revision and the content will still be published on the site. I think that that is quite dangerous behaviour.